pkg/kubectl/cmd/util/helpers.go

```diff
@@ -122,6 +122,11 @@ func checkErrWithPrefix(prefix string, err error) {
 // checkErr formats a given error as a string and calls the passed handleErr
 // func with that string and an kubectl exit code.
 func checkErr(prefix string, err error, handleErr func(string, int)) {
+	// unwrap aggregates of 1
+	if agg, ok := err.(utilerrors.Aggregate); ok && len(agg.Errors()) == 1 {
```

Comment:
I was rather thinking about a usecase when you're creating an aggregate from several errors and one of them is an aggregate already. You'd still create an aggregate but instead of:
`aggregate[err, err, aggregate(err)]` you could have `aggregate[err, err, err]`. The 3rd error would be unpacked upon creating new aggregate. Does that make sense?